product_pricelist_margins/models/product_pricelist_item.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+
+from odoo import api, fields, models
+from odoo.tools.float_utils import float_is_zero
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+
+    cost = fields.Float(
+        related="product_tmpl_id.standard_price",
+        digits="Product Price",
+    )
+    margin = fields.Float(
+        compute="_compute_margin",
+        digits="Product Price",
+    )
+    margin_percent = fields.Float(
+        string="Margin (%)",
+        compute="_compute_margin",
+    )
+
+    @api.depends("fixed_price", "product_id", "min_quantity", "cost")
+    def _compute_margin(self):
+        for item in self:
+            margin = percentage = 0
+            if (
+                not float_is_zero(
+                    item.fixed_price, precision_digits=item.currency_id.rounding
+                )
+                and self.min_quantity
+            ):
+                res = item.product_id.taxes_id.compute_all(
+                    item.fixed_price,
+                    item.currency_id,
+                    item.min_quantity,
+                    product=item.product_id,
+                )
+
+                price_vat_excl = res["total_excluded"] / item.min_quantity
```

Comment:
```suggestion
                price_vat_excl = res["total_excluded"]
```
don't care item.min_quantity, from my previous comment, the price is computed by qty=1 by default